.eslintrc.json

```diff
@@ -31,6 +31,8 @@
     }
   },
   "rules": {
+    "import/namespace": "off",
+    "import/named": "off",
```

Comment:
curious what was the error.

Reply:
it's related to vitest ./utils import! so i solved it with thsi as suggested.

Reply:
I'll revisit it later.